src/database/models/triprequest.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const tripRequest = sequelize.define('tripRequest', {
+    leavingFrom: DataTypes.STRING,
+    goingTo: DataTypes.INTEGER,
+    travelDate: DataTypes.STRING,
+    returnDate: DataTypes.STRING,
+    travelReason: DataTypes.STRING,
+    tripType: DataTypes.STRING,
+    status: DataTypes.STRING,
+    accomodationId: DataTypes.INTEGER
+  }, {});
+  tripRequest.associate = function (models) {
+    tripRequest.belongsTo(models.users, {
+      foreignKey: 'requesterId',
+      onDelete: 'CASCADE',
```

Comment:
It should be associated with the accommodation (it belongsTo accommodation)

Reply:
do I have to include accomodation model into tripRequest feature or we wait for accomodation feature to be marged?